src/vs/workbench/api/common/positron/extHostLanguageRuntime.ts

```diff
@@ -285,7 +287,9 @@ export class ExtHostLanguageRuntime implements extHostProtocol.ExtHostLanguageRu
 			// Amend the message with the event clock for ordering
 			const runtimeMessage: ILanguageRuntimeMessage = {
 				event_clock: tick,
-				...message
+				...message,
+				// Wrap buffers in VSBuffer so that they can be sent to the main thread
+				buffers: message.buffers?.map(buffer => VSBuffer.wrap(buffer)),
```

Comment:
Technically this will add to the object to always have a key of 'buffers' with a value of `undefined`, which is different than an object without the key of `'buffers'`. Even though both return `undefined` accessed with `obj['buffers']`.  E.g. 

```js
let objWithUndefined = { myKey: undefined };
let objWithoutKey = {};

console.log(objWithUndefined.myKey); // undefined
console.log(objWithoutKey.myKey); // undefined

console.log(objWithUndefined.hasOwnProperty('myKey')); // true
console.log(objWithoutKey.hasOwnProperty('myKey')); // false

console.log(Object.keys(objWithUndefined)); // ['myKey']
console.log(Object.keys(objWithoutKey)); // []
```
How this info is serialized I don't think is standardized either.